README.md

```diff
@@ -1,519 +1,136 @@
+<p align="center">
+  <br/>
+    <img alt="lighteval library logo" src="./assets/lighteval-doc.svg" width="376" height="59" style="max-width: 100%;">
```

Comment:
Are we removing cloud + sun as our logo for lighteval?

Reply:
I felt like it would have been too crowded with the HF logo

Reply:
I think it's nice when libs have their own logo, like :firecracker: for diffusers, etc, but OK for now